### PR TITLE
Collisions

### PR DESCRIPTION
**Pull request by ercastil**
Changes were made to:

1) **SQLQuery**: 

- **get_or_create**: try to add an object and commit the change, if the object already exists perform a rollback and return the current value 
- **bulk_insert**: insert a list of objects using the engine.execute() and sqlalchemy.dialects.postgresql , a condition insert, if objects already exists do nothing

2) **setup.py**: 

- a change was made to the line:
packages=find_namespace_packages(include=['db_plugins.*'] to look for "db_plugins.*" sub modules instead of "plugins.*"

### CODE REVIEW

File: db_plugins/db/sql/__init__.py [hunk 1]
```diff
@@ -50,30 +51,34 @@ def get_or_create(self, model=None, filter_by=None, **kwargs):
...
+
+        try:
+            kwargs.update(filter_by)
+            instance = model(**kwargs)
             self.session.add(instance)
-            return instance, True
+            self.session.commit()
+            result = instance
+            created = True
+        except:
```
Comment (cvalenzu) [inline db_plugins/db/sql/__init__.py:66]: Maybe just catch the duplicate object exception

File: db_plugins/db/sql/__init__.py [hunk 2]
```diff
@@ -50,30 +51,34 @@ def get_or_create(self, model=None, filter_by=None, **kwargs):
...
+            self.session.rollback()
+            result = self.session.query(model).filter_by(**filter_by).first()
+            created = False
```
Comment (cvalenzu) [inline db_plugins/db/sql/__init__.py:69]: Maybe just catch the duplicate object exception (`IntegrityError`)

**Review by cvalenzu — CHANGES_REQUESTED** [review]

File: db_plugins/db/sql/__init__.py (hunk 2 above)
Comment (ercastil) [reply db_plugins/db/sql/__init__.py]: changed

Comment (ercastil) [reply db_plugins/db/sql/__init__.py]: from sqlalchemy.exc import IntegrityError added

**Pull request merged**